pkg/operator/ceph/csi/template/cephfs/csi-cephfsplugin-provisioner-dep.yaml

```diff
@@ -33,14 +34,14 @@ spec:
             - "--leader-election-lease-duration=137s"
             - "--leader-election-renew-deadline=107s"
             - "--leader-election-retry-period=26s"
-            - "--default-fstype=ext4"
```

Comment:
Why remove the default fstype?

Reply:
fs no need to assign type

Reply:
fstype is required only for rbd as we need to format the rbd device and mount it to the pods for cephfs we dont need to do it for that reason we can remove it.